cf/env_test.go

```diff
@@ -46,7 +47,7 @@ var _ = Describe("CF Env", func() {
 		Expect(os.Setenv("VCAP_APPLICATION", vcapApplication)).ShouldNot(HaveOccurred())
 		Expect(os.Setenv("VCAP_SERVICES", "{}")).ShouldNot(HaveOccurred())
 
-		environment, err = env.New(pflag.CommandLine)
+		environment, err = env.New(context.TODO(), pflag.CommandLine)
```

Comment:
This test doesn't seem valid. `env.New` isn't used anywhere in this project. Maybe it needs to be changed to `cf.DefaultEnv` (also, the package needs `_test`)